### PR TITLE
Add XMPlay 3.8.3.20 (Stuff Version)

### PR DESCRIPTION
**Pull request by Retia-Adolf**
i.e. pre-release version

### CODE REVIEW

File: bucket/xmplay-stuff.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "xmplay.ini",
+        "xmplay.library",
+        "xmplay.pls",
+        "user-plugins",
+        "xmp-skin"
+    ],
+    "checkver": {
+        "url": "https://www.un4seen.com/xmplay-stuff.php",
+        "regex": "([\\d.]+)</span>"
+    }
```
Comment (Ash258) [inline bucket/xmplay-stuff.json]: checkver without autoupdate??

Comment (Retia-Adolf) [reply bucket/xmplay-stuff.json]: hmmm, it has the same file name for latest version.

Comment (Ash258) [reply bucket/xmplay-stuff.json]: And where is said that it works this way?

Autoupdate property needs to be specified. (As I explained to you already https://github.com/lukesampson/scoop-extras/pull/1886#issuecomment-474809744)

**Comment by Ash258** [comment]
Use fixed manifest from PR in extras bucket

File: bucket/xmplay-stuff.json (hunk 1 above)
Comment (Retia-Adolf) [reply bucket/xmplay-stuff.json]: Ok

File: bucket/xmplay-stuff.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "3.8.3.20",
+    "license": "freeware",
+    "homepage": "https://www.un4seen.com/xmplay.html",
+    "description": "The latest (potentially unstable) build of XMPlay, a Win32 audio player. In the Un4seen forum, it is refferred to as the \"stuff version\".",
```
Comment (Ash258) [inline bucket/xmplay-stuff.json]:
```suggestion
    "description": "The latest (potentially unstable) build of XMPlay, a Win32 audio player.",
```

File: bucket/xmplay-stuff.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "url": "https://www.un4seen.com/stuff/xmplay.exe",
+    "hash": "sha512:ef9752857d9a9dc71b4d2bf04b4f2cd7f37fe14282d5cbc7d8e7800cd4b1074540333087c4e3163a74fd05f6e05e6d87011496f62a3cce9420c2ed3d9f1f9046",
```
Comment (Ash258) [inline bucket/xmplay-stuff.json]: Run checkver to correctly calculate hashes

File: bucket/xmplay-stuff.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "3.8.3.20",
+    "license": "freeware",
```
Comment (Ash258) [inline bucket/xmplay-stuff.json]:
```suggestion
    "license": "Freeware",
```

File: bucket/xmplay-stuff.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    "pre_install": [
+        "# Create placeholder files",
+        "function CreateFile([String[]] $file) {",
+        "    foreach ($f in $file) {",
+        "        if (! (Test-Path \"$dir\\$f\")) { New-Item \"$dir\\$f\" -ItemType 'File' | Out-Null }",
+        "    }",
+        "}",
+        "CreateFile 'xmplay.ini', 'xmplay.library', 'xmplay.pls'"
+    ],
+    "bin": "xmplay.exe",
```
Comment (Ash258) [inline bucket/xmplay-stuff.json]:
```suggestion
    "bin": [
        [
            "xmplay.exe",
            "xmplay-stuff"
        ]
    ],
```

**Pull request merged**